src/ocgis/conv/nc.py

```diff
@@ -29,6 +37,9 @@ class NcConverter(AbstractCollectionConverter):
     +------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
     | unlimited_to_fixedsize | If ``True``, convert the unlimited dimension to fixed size. Only applies to time and level dimensions.                                 |
     +------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
+    | geom_dim               | The name of the dimension storing aggregated (unioned) outputs. Only applies when ``aggregate is True``.                               |
+    +------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
What do you think about adding an optional keyword argument defining an attribute on the geometry unique identifier that provides a name, path, etc. to the source geometry data? Maybe it makes more sense to put this info on the history attribute.

Reply:
I think that would be smart. The history attribute often ends up very crowded, I suggest saving that information into a global attribute. Could that be done automatically, that is without entering the information in the options but just by parsing the geom information ?

Reply:
It could be done with a little work to the geometry input (maintaining the path for example). However, there are many cases where the geometry will be a bare shapely geometry or bounding box. Also, one might not want to append a full path to the geometry file, etc. I think it should be something provided by the user or, at the very least, turned off. Can do some thinking on what would be best and can definitely add it later.

Reply:
Right. Note that I'm copying over the geom attributes into netCDF variables, so that's another place the info could appear.